spec/test_app/db/schema.rb

```diff
@@ -56,6 +65,8 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "slug"
+    t.text     "about"
+    t.string   "banner_url"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.